gnucash/import-export/aqb/gnc-gwen-gui.c

```diff
@@ -1035,6 +1058,43 @@ get_input(GncGWENGui *gui, guint32 flags, const gchar *title, const gchar *text,
         g_free(raw_text);
     }
 
+    #ifdef AQBANKING6
+    //if (optical_challenge)
+    if(mimeType != NULL && pChallenge != NULL && lChallenge > 0)
+    {
+        // convert PNG and load into widget
+        // TBD: check mimeType?
+        guchar *gudata = (guchar*)pChallenge;
+
+        GError *error = NULL;
+        GdkPixbufLoader *loader = gdk_pixbuf_loader_new_with_mime_type(mimeType, &error);
+        GdkPixbuf *pixbuf;
+
+        if(error != NULL)
+        {
+            printf("Error; perhaps mimeType (%s) not supported?\n%s\n", mimeType, error->message);
```

Comment:
Wow, looked right past that. Messages should go to the logger, e.g.
```
PERR("Pixbuf loader not loaded: %s, perhaps MIME type %s isn't supported.", error->message, mimeType);
```

Reply:
Okay, changed that and removed the other printf's. How stringent should error handling be, by the way?